app/decorators/account_decorator.rb

```diff
@@ -3,9 +3,30 @@
 class AccountDecorator < ApplicationDecorator
   delegate :display_name, :email, to: :admin_user, prefix: true
 
+  self.include_root_in_json = false
+
+  def new_application_data
+    {
+      id: id.to_s,
+      name: name,
+      description: "Admin: #{admin_user_display_name}",
+      createdAt: created_at.to_s(:long),
+      contractedProducts: contracts,
+      createApplicationPath: h.admin_buyers_account_applications_url(object),
```

Comment:
I would use `path` instead of `url` it is generally accepted convention that website use relative path to their domain instead of including domain
```suggestion
      createApplicationPath: h.admin_buyers_account_applications_path(object),
```